engine/gamesys/src/gamesys/scripts/script_sound.cpp

```diff
@@ -535,13 +534,14 @@ namespace dmGameSystem
             lua_pop(L, 1);
         }
 
+        uint32_t play_id = dmSound::GetAndIncreasePlayCounter();
```

Comment:
Before, we use play_id only for sounds with callback. That was the reason why we don't ever send messages. I removed this condition (see comment above) and also we need `play_id` for each sound to be able to stop it separately.